mods/ra/rules/ai.yaml

```diff
@@ -385,6 +526,8 @@ Player:
 			Production: barr,tent,weap,afld,hpad
 			NavalProduction: spen,syrd
 			Silo: silo
+			Fragile: atek,stek,afld,hpad,mslo,iron,pdox,silo # These are build at safer positions.
```

Comment:
Not sure about domes, they are good detectors too. Not everything can be hidden at the back of the base. Something has to stand in front!